docs/API/helpers/dynamic-components.md

```
@@ -74,6 +228,185 @@ Fliplet.Helper({
 
 ## Repeater
 
+# List Repeater JS API
+
+### `Fliplet.ListRepeater.get(filter, options)`
+
+```js
+Fliplet.ListRepeater.get(filter, options)
+```
+
+- **filter** (`String|Object`)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 0

(MD007, ul-indent)